python/interpret_community/mimic/models/lightgbm_model.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, multiclass=False, random_state=DEFAULT_RANDOM_STATE,
                             '\nOtherwise, if multiclass=False, uses the parameters for LGBMRegressor:\n' +
                             LGBMRegressor.__init__.__doc__)
     except Exception:
-        print("Could not import lightgbm, required if using LGBMExplainableModel")
+        pass
```

Comment:
Just to be sure, the idea is that we'll catch issues on line 26 and that's why we don't need to print here?

Reply:
yep, exactly right!